keycloak/keycloak-ui
# Issue 2765

## Motivation
Closes #2765 

## Verification Steps
1. Go to `Authentication >> Flows`.
2. Verify that the items are now also listed alphabetically as per #2765. Previously items were listed only by type, but now they're also listed alphabetically.

## Checklist:
